db-service/lib/cqn2sql.js

```diff
@@ -90,10 +81,17 @@ class CQN2SQLRenderer {
     if (vars?.length && !this.values?.length) this.values = vars
     if (vars && Object.keys(vars).length && !this.values?.length) this.values = vars
     const sanitize_values = process.env.NODE_ENV === 'production' && cds.env.log.sanitize_values !== false
-    DEBUG?.(
-      this.sql,
-      ...(sanitize_values && (this.entries || this.values?.length > 0) ? ['***'] : this.entries || this.values || []),
-    )
+
+    const DEBUG_ENABLED = cds.log('sql')._debug || cds.log('sqlite')._debug
```

Comment:
calling `cds.log` this often will make it much more heavy then just always calling `DEBUG`. Just make these two `cds.log`s at the start of the file.